app/views/articles/index.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Articles
+
+- @articles.each do |article|
+    %img{:src => "/assets/article.jpg"}/
+    %h3= article.title
+    %p= article.body
```

Comment:
Why are you displaying the full body of an article on this index?

Reply:
This is the MVP. We'll hide the full body when we get to paywall story.